src/WSDecoder.elm

```diff
@@ -107,6 +107,33 @@ type PlayerObj
     | PlayerB Int PlayerType PType
 
 
+addmediatype : PType -> ItemDetails -> ItemDetails
+addmediatype mediaType details =
+    { details | mediatype = mediaType }
+
+
+getmediatype : List PlayerObj -> ItemDetails -> ItemDetails
+getmediatype player itemdetail =
+    let
+        x =
```

Comment:
you probably don't need this let anymore? should be simple enought to do it inline